sdk/turing/router/config/router_version.py

```diff
@@ -47,8 +47,9 @@ def __init__(
         self.status = RouterStatus(status)
         self.name = name
         self.monitoring_url = monitoring_url
-        self.log_config = RouterVersionLogConfig(**kwargs.get("log_config"))
```

Comment:
because `LogConfig` validates against the `Protocol`, the `RouterConfig` has to be initiated first